tests/MongoDB.Driver.Core.TestHelpers/ClusterDescriptionParser.cs

```diff
@@ -0,0 +1,60 @@
+﻿/* Copyright 2018-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using MongoDB.Bson;
+using MongoDB.Driver.Core.Clusters;
+using MongoDB.Driver.Core.Servers;
+
+namespace MongoDB.Driver.Core.TestHelpers
+{
+    public static class ClusterDescriptionParser
+    {
+        public static ClusterDescription Parse(BsonDocument args)
```

Comment:
Doesn't cover all possible values, just those needed for testing so far.

This can be enhanced as necessary in the future.

Note also that defaults are used for some values to make creating test values easier.